plugins/modules/ovirt_host_network.py

```diff
@@ -579,7 +579,7 @@ def main():
                     ] if labels else None,
                     removed_network_attachments=attachments if attachments else None,
                 )
-                if engine_supported(connection, '4.3'):
+                if engine_supported(connection, '4.4'):
```

Comment:
Please add a comment: ```# TODO: Remove after https://bugzilla.redhat.com/1798818 is fixed

Reply:
I would keep it as it is if there was issue in 4.3 and will get fixed in 4.4. It would be better to use the older method of the save configuration `host_networks_module._action_save_configuration`.

Reply:
OK

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1798818 is included in oVirt 4.4.1, let's merge it